docs/changelog.md

```diff
@@ -13,6 +13,10 @@ permalink: /changelog/
 * **feature-customtabs**
  * Sharing a URL from a custom tab always uses the current url of the session. [bug #1831803](https://bugzilla.mozilla.org/show_bug.cgi?id=1831803)
 
+* **browser-toolbar**
+    * Add `showMenuButton` and `hideMenuButton` API to `BrowserToolbar` and `DisplayToolbar` to allow hiding and showing of the menu button in
+    the `BrowserToolbar` [Bug 1864760](https://bugzilla.mozilla.org/show_bug.cgi?id=1864760)
```

Comment:
Let's land the AC change directly in `main`. We will also need to update this to be under 124.0

Reply:
new PR up, we'll wait for that one to make it to `main` and then I'll rebase the dev branch and clean up this branch.

Reply:
This PR now contains only the Fenix changes, AC changes have been merged into `main`.